charts/flyteagent/values.yaml

```diff
@@ -56,6 +56,12 @@ serviceAccount:
   imagePullSecrets: []
 # -- Security context for pod
 podSecurityContext: {}
+# -- Readiness probe for flyteagent. Use readinessProbe: {} if agent doesn't implement grpc-health-checking service.
+# -- https://kubernetes.io/blog/2022/05/13/grpc-probes-now-in-beta/#trying-the-feature-out
+readinessProbe:
+  grpc:
+    port: 8000
+  initialDelaySeconds: 5
```

Comment:
Why is it that high? or why is it needed?

Reply:
importing some python modules is very slow, especially airflow and spark.